lib/matplotlib/mathtext.py

```diff
@@ -74,10 +75,12 @@ def parse(self, s, dpi=72, prop=None):
         # is mutable; key the cache using an internal copy (see
         # text._get_text_metrics_with_cache for a similar case).
         prop = prop.copy() if prop is not None else None
-        return self._parse_cached(s, dpi, prop)
+        if antialiased is None:
+            antialiased = mpl.rcParams['text.antialiased']
+        return self._parse_cached(s, dpi, prop, antialiased)
 
     @functools.lru_cache(50)
-    def _parse_cached(self, s, dpi, prop):
+    def _parse_cached(self, s, dpi, prop, antialiased):
```

Comment:
This also fixes a latent bug where we would incorrectly hit the cache if the rcparam was changed between the first and subsequent calls!